abra2_2.22/abra2_2.22.cwl

```diff
@@ -44,7 +44,7 @@ inputs:
       prefix: '--ref'
     doc: Genome reference location
     secondaryFiles:
-      - ^.fai
+      - ^.fasta.fai
```

Comment:
This depends on the name of the reference file you are using, if the name is `hg19.fasta` and the `hg19.fasta.fai` is fai then `^.fai` should work.

Reply:
@murphycj2  please let me know if that is not the case

Reply:
@rhshah  that is what i thought, but using just `^.fai` did not work for me. Happened before to me. i am no expert in toil/cwl, so I am not sure what is going on.

Reply:
I think the ^ means "remove the last part of the suffix after '.'" 

So probably just ".fai" should work

Reply:
removing the ^ works, thanks @ionox0 !